ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/router-multicast.yml

```diff
@@ -2,6 +2,13 @@
 
 router_multicast:
   ipv4:
+    rpf:
+      routes:
+        - subnet: 10.10.10.1/32
```

Comment:
Sorry to bother again. I just tested these commands. I found that same subnet can have multiple nexthops.
<img width="281" alt="image" src="https://user-images.githubusercontent.com/88734745/161996131-c5c0ad63-0545-455d-b264-0c2987a75ba2.png">
<img width="281" alt="image" src="https://user-images.githubusercontent.com/88734745/161996131-c5c0ad63-0545-455d-b264-0c2987a75ba2.png">